SJ - Sparc Cloud Branch

Stuart-Johnson opened the pull request:
Listing out stories below:

[#168904200]

kayla-glick commented on db/migrate/20191101150844_add_ontology_tag_system.rb:
```diff
@@ -0,0 +1,29 @@
+    add_column :organizations, :ontology_tag, :string
+
+    ## Populate new ontology tag permissible values
+    unless PermissibleValue.where(category: 'ontology_tag').exists?
```
Instead of creating the PermissibleValues here, can you add a source CSV file and run the import task?

kayla-glick committed:
fix dotenv.example org_logo name

SamWord committed:
protocol_soap_endpoints_controller accepts RPE SOAP message - no data parsing yet

SamWord committed:
Added lib file to make test calls to the OnCore endpoint in development. Do not keep.

dependabot-preview[bot] committed:
Bump launchy from 2.4.3 to 2.5.0

Bumps [launchy](https://github.com/copiousfreetime/launchy) from 2.4.3 to 2.5.0.
- [Release notes](https://github.com/copiousfreetime/launchy/releases)
- [Changelog](https://github.com/copiousfreetime/launchy/blob/master/README.md)
- [Commits](https://github.com/copiousfreetime/launchy/compare/v2.4.3...v2.5.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2236 from sparc-request/dependabot/bundler/acts_as_list-a8a755c

Bump acts_as_list from `d3172df` to `a8a755c`

Stuart-Johnson committed:
Merge pull request #2238 from sparc-request/dependabot/bundler/launchy-2.5.0

Bump launchy from 2.4.3 to 2.5.0

kayla-glick committed:
remove indexes from sharded foreign keys

dependabot-preview[bot] committed:
[Security] Bump puma from 4.3.1 to 4.3.3

Bumps [puma](https://github.com/puma/puma) from 4.3.1 to 4.3.3. **This update includes security fixes.**
- [Release notes](https://github.com/puma/puma/releases)
- [Changelog](https://github.com/puma/puma/blob/master/History.md)
- [Commits](https://github.com/puma/puma/compare/v4.3.1...v4.3.3)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2218 from sparc-request/kg-organization_deep_link

KG - Organization Deep Link

Stuart-Johnson committed:
Merge pull request #2239 from sparc-request/dependabot/bundler/puma-4.3.3

[Security] Bump puma from 4.3.1 to 4.3.3

kayla-glick committed:
merge conflicts

kayla-glick committed:
reconfigure API and remote service notifier to use sharding

dependabot-preview[bot] committed:
[Security] Bump omniauth from 1.9.0 to 1.9.1

Bumps [omniauth](https://github.com/omniauth/omniauth) from 1.9.0 to 1.9.1. **This update includes a security fix.**
- [Release notes](https://github.com/omniauth/omniauth/releases)
- [Commits](https://github.com/omniauth/omniauth/compare/v1.9.0...v1.9.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

kayla-glick committed:
[#170957314] fix authorization of calendar structure table

kayla-glick committed:
fix service accordion bad merge conflicts

kayla-glick committed:
add styling context for external requests

SamWord committed:
Renamed endpoint controller, WSDL will generate and seems to properly accept SOAP messages [#170872411]

SamWord committed:
Renamed endpoint controller file to match controller class name [#170872411]

Stuart-Johnson committed:
Merge pull request #2241 from sparc-request/dependabot/bundler/omniauth-1.9.1

[Security] Bump omniauth from 1.9.0 to 1.9.1

SamWord committed:
Added logger for OnCore-{environment}.log to view accepted params in -d [#170872411]

SamWord committed:
Removed local test interface

SamWord committed:
Fixed merge conflicts

SamWord committed:
fully resolved merge conflicts

dependabot-preview[bot] committed:
Bump acts_as_list from `a8a755c` to `a41c31e`

Bumps [acts_as_list](https://github.com/swanandp/acts_as_list) from `a8a755c` to `a41c31e`.
- [Release notes](https://github.com/swanandp/acts_as_list/releases)
- [Commits](https://github.com/swanandp/acts_as_list/compare/a8a755cc13545b92252a234245480813c79519cb...a41c31eafc9c1d952b5e97477fa01c52f58d2b18)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2243 from sparc-request/saw-oncore-rpe-endpoint

SAW - OnCore SOAP Endpoint

dependabot-preview[bot] committed:
Bump nokogiri from 1.10.8 to 1.10.9

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.10.8 to 1.10.9.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/v1.10.9/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.10.8...v1.10.9)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

SamWord committed:
Re-added local test interface

kayla-glick committed:
update service/organization deep link URLs

kayla-glick committed:
fixing delayed job hopefully

SamWord committed:
Added timestamp to epic log

Stuart-Johnson committed:
Merge pull request #2240 from sparc-request/dependabot/bundler/nokogiri-1.10.9

Bump nokogiri from 1.10.8 to 1.10.9

Stuart-Johnson committed:
Merge pull request #2244 from sparc-request/dependabot/bundler/acts_as_list-a41c31e

Bump acts_as_list from `a8a755c` to `a41c31e`

Stuart-Johnson committed:
Merge pull request #2242 from sparc-request/kg-calendar_structure_bug

KG - Calendar Bugs

Stuart-Johnson committed:
Merge pull request #2245 from sparc-request/saw-epic-soap-timestamp

SAW - Added Timestamp to Epic SOAP Log

kayla-glick committed:
add exernal request language to submission emails

kayla-glick committed:
hide non-externally shared services in catalog when logged in

kayla-glick committed:
add external sharing text to homepage

kayla-glick committed:
updating some email config

kayla-glick committed:
fix devise initializer

SamWord committed:
Don't count "" as a state to determine which surveys/forms to hide in filter dropdown

SamWord committed:
Autocomplete off makes it so filter displays the actual filter value

dependabot-preview[bot] committed:
Bump capistrano from 3.12.0 to 3.12.1

Bumps [capistrano](https://github.com/capistrano/capistrano) from 3.12.0 to 3.12.1.
- [Release notes](https://github.com/capistrano/capistrano/releases)
- [Commits](https://github.com/capistrano/capistrano/compare/v3.12.0...v3.12.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump rack-mini-profiler from 1.1.6 to 2.0.0

Bumps [rack-mini-profiler](https://github.com/MiniProfiler/rack-mini-profiler) from 1.1.6 to 2.0.0.
- [Release notes](https://github.com/MiniProfiler/rack-mini-profiler/releases)
- [Changelog](https://github.com/MiniProfiler/rack-mini-profiler/blob/master/CHANGELOG.md)
- [Commits](https://github.com/MiniProfiler/rack-mini-profiler/compare/v1.1.6...v2.0.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump rspec-rails from 3.9.0 to 3.9.1

Bumps [rspec-rails](https://github.com/rspec/rspec-rails) from 3.9.0 to 3.9.1.
- [Release notes](https://github.com/rspec/rspec-rails/releases)
- [Changelog](https://github.com/rspec/rspec-rails/blob/master/Changelog.md)
- [Commits](https://github.com/rspec/rspec-rails/compare/v3.9.0...v3.9.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump webmock from 3.8.2 to 3.8.3

Bumps [webmock](https://github.com/bblimke/webmock) from 3.8.2 to 3.8.3.
- [Release notes](https://github.com/bblimke/webmock/releases)
- [Changelog](https://github.com/bblimke/webmock/blob/master/CHANGELOG.md)
- [Commits](https://github.com/bblimke/webmock/compare/v3.8.2...v3.8.3)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2249 from sparc-request/dependabot/bundler/rspec-rails-3.9.1

Bump rspec-rails from 3.9.0 to 3.9.1

Stuart-Johnson committed:
Merge pull request #2247 from sparc-request/dependabot/bundler/capistrano-3.12.1

Bump capistrano from 3.12.0 to 3.12.1

Stuart-Johnson committed:
Merge pull request #2248 from sparc-request/dependabot/bundler/rack-mini-profiler-2.0.0

Bump rack-mini-profiler from 1.1.6 to 2.0.0

Stuart-Johnson committed:
Merge pull request #2250 from sparc-request/dependabot/bundler/webmock-3.8.3

Bump webmock from 3.8.2 to 3.8.3

kayla-glick committed:
#171602442] fix ssr creation and ssr_id assignment code

kayla-glick committed:
[#167900874] add funding lock

dependabot-preview[bot] committed:
Bump grape from 1.3.0 to 1.3.1

Bumps [grape](https://github.com/ruby-grape/grape) from 1.3.0 to 1.3.1.
- [Release notes](https://github.com/ruby-grape/grape/releases)
- [Changelog](https://github.com/ruby-grape/grape/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ruby-grape/grape/compare/v1.3.0...v1.3.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

kayla-glick committed:
fixing delayed job in production mode

kayla-glick committed:
fix subsidy header

kayla-glick committed:
additional fixes for delayed job external emails

SamWord committed:
added SOAP response, no parsing yet

kayla-glick committed:
fixing delayed job loading with wrong shard

Stuart-Johnson committed:
Merge pull request #2253 from sparc-request/dependabot/bundler/grape-1.3.1

Bump grape from 1.3.0 to 1.3.1

Stuart-Johnson committed:
Merge pull request #2246 from sparc-request/saw-filter-bug

SAW - SPARCForms Filter Bug

Stuart-Johnson committed:
Merge pull request #2252 from sparc-request/kg-funding_lock

KG - Add Funding Lock

Stuart-Johnson committed:
Merge pull request #2255 from sparc-request/saw-oncore-rpe-endpoint

SAW - OnCore Endpoint SOAP Response

Commit:
updating deploy file

kayla-glick committed:
fix unit type gsub bug

kayla-glick committed:
fix API and service/org deep links

kayla-glick committed:
[#171602442] add check on ssr update ssr_id

dependabot-preview[bot] committed:
Bump parallel_tests from 2.31.0 to 2.32.0

Bumps [parallel_tests](https://github.com/grosser/parallel_tests) from 2.31.0 to 2.32.0.
- [Release notes](https://github.com/grosser/parallel_tests/releases)
- [Commits](https://github.com/grosser/parallel_tests/compare/v2.31.0...v2.32.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump rack-mini-profiler from 2.0.0 to 2.0.1

Bumps [rack-mini-profiler](https://github.com/MiniProfiler/rack-mini-profiler) from 2.0.0 to 2.0.1.
- [Release notes](https://github.com/MiniProfiler/rack-mini-profiler/releases)
- [Changelog](https://github.com/MiniProfiler/rack-mini-profiler/blob/master/CHANGELOG.md)
- [Commits](https://github.com/MiniProfiler/rack-mini-profiler/compare/v2.0.0...v2.0.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

kayla-glick committed:
add missing method

kayla-glick committed:
remove unused method / speec

kayla-glick committed:
remove ssr_has_changed?

SamWord committed:
Overrided SOAP response template to fir OnCore requirements

dependabot-preview[bot] committed:
Bump premailer-rails from 1.10.3 to 1.11.0

Bumps [premailer-rails](https://github.com/fphilipe/premailer-rails) from 1.10.3 to 1.11.0.
- [Release notes](https://github.com/fphilipe/premailer-rails/releases)
- [Changelog](https://github.com/fphilipe/premailer-rails/blob/master/CHANGELOG.md)
- [Commits](https://github.com/fphilipe/premailer-rails/compare/v1.10.3...v1.11.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

kayla-glick committed:
fix spec

Stuart-Johnson committed:
Merge pull request #2258 from sparc-request/saw-oncore-rpe-endpoint

SAW - Overrided SOAP Response Template

heathharrelson committed:
primary PI lookup with lazy_load_ldap [#171888331]

Ensure idempotent primary PI identity lookup in `protocol_params`
when lazy_load_ldap is enabled.

kayla-glick committed:
fix ssr_id assignment to before_validation

chrisortman committed:
Removes invalid xplat characters from file names

dependabot-preview[bot] committed:
Bump pry from 0.12.2 to 0.13.0

Bumps [pry](https://github.com/pry/pry) from 0.12.2 to 0.13.0.
- [Release notes](https://github.com/pry/pry/releases)
- [Changelog](https://github.com/pry/pry/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pry/pry/compare/v0.12.2...v0.13.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2263 from sparc-request/dependabot/bundler/pry-0.13.0

Bump pry from 0.12.2 to 0.13.0

Stuart-Johnson committed:
Merge pull request #2257 from sparc-request/dependabot/bundler/rack-mini-profiler-2.0.1

Bump rack-mini-profiler from 2.0.0 to 2.0.1

kayla-glick committed:
[#171773981] don't change ssr status whn updating subject count in admin edit

kayla-glick committed:
[#170752951] better max subject count handling

dependabot-preview[bot] committed:
Bump faker from 2.10.2 to 2.11.0

Bumps [faker](https://github.com/faker-ruby/faker) from 2.10.2 to 2.11.0.
- [Release notes](https://github.com/faker-ruby/faker/releases)
- [Changelog](https://github.com/faker-ruby/faker/blob/master/CHANGELOG.md)
- [Commits](https://github.com/faker-ruby/faker/compare/v2.10.2...v2.11.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump rspec-rails from 3.9.1 to 4.0.0

Bumps [rspec-rails](https://github.com/rspec/rspec-rails) from 3.9.1 to 4.0.0.
- [Release notes](https://github.com/rspec/rspec-rails/releases)
- [Changelog](https://github.com/rspec/rspec-rails/blob/master/Changelog.md)
- [Commits](https://github.com/rspec/rspec-rails/compare/v3.9.1...v4.0.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

kayla-glick committed:
spec fixes

kayla-glick committed:
[#169095503] remove rmid validated when RMID removed

Stuart-Johnson committed:
Merge pull request #2267 from sparc-request/dependabot/bundler/rspec-rails-4.0.0

Bump rspec-rails from 3.9.1 to 4.0.0

Stuart-Johnson committed:
Merge pull request #2266 from sparc-request/dependabot/bundler/faker-2.11.0

Bump faker from 2.10.2 to 2.11.0

Stuart-Johnson committed:
Merge pull request #2260 from OCTRI/lazy-load-ldap-fixes

HH - (SPARCRequest) Errors With lazy_load_ldap Enabled [#171888331]

dependabot-preview[bot] committed:
Bump ruby-prof from 1.3.0 to 1.3.1

Bumps [ruby-prof](https://github.com/ruby-prof/ruby-prof) from 1.3.0 to 1.3.1.
- [Release notes](https://github.com/ruby-prof/ruby-prof/releases)
- [Changelog](https://github.com/ruby-prof/ruby-prof/blob/master/CHANGES)
- [Commits](https://github.com/ruby-prof/ruby-prof/compare/1.3.0...1.3.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump rails from 5.2.4.1 to 5.2.4.2

Bumps [rails](https://github.com/rails/rails) from 5.2.4.1 to 5.2.4.2.
- [Release notes](https://github.com/rails/rails/releases)
- [Commits](https://github.com/rails/rails/compare/v5.2.4.1...v5.2.4.2)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2262 from ui-icts/question-mark-renames

Removes invalid xplat characters from file names

Stuart-Johnson committed:
Merge pull request #2268 from sparc-request/kg-rmid_verified

KG - RMID Validated Flag When RMID Removed

kayla-glick committed:
[#169156333] add ssr resubmission check modal

kayla-glick committed:
spc fixes

kayla-glick committed:
trigger travis build

SamWord committed:
Merge remote-tracking branch 'origin/kg-duplicate_ssr_bug' into saw-oncore-rpe-endpoint

Stuart-Johnson committed:
Merge branch 'v3.7.0' into kg-duplicate_ssr_bug

kayla-glick committed:
I really do love specs

kayla-glick committed:
Merge branch 'kg-duplicate_ssr_bug' of https://github.com/sparc-request/sparc-request into kg-duplicate_ssr_bug

jleonardw9 committed:
Cleaning up merge column display, always showing the column for now.

jleonardw9 committed:
Creating an association between protocols and protocol merges.

Stuart-Johnson committed:
Merge branch 'master' into v3.7.0

kayla-glick committed:
fixes and specs

kayla-glick committed:
spec fixes AGAIN

SamWord committed:
Merge remote-tracking branch 'origin/kg-duplicate_ssr_bug' into saw-oncore-rpe-endpoint

kayla-glick committed:
not incrementing protocol.next_ssr_id

SamWord committed:
Merge remote-tracking branch 'origin/kg-duplicate_ssr_bug' into saw-oncore-rpe-endpoint

kayla-glick committed:
fix bug

SamWord committed:
Merge remote-tracking branch 'origin/kg-duplicate_ssr_bug' into saw-oncore-rpe-endpoint

kayla-glick committed:
fix bug

SamWord committed:
resolved merge conflict

SamWord committed:
Example SOAP message can be parsed and viewed in SPARC (basic functionality) [#170872411]

heathharrelson committed:
Cache Available Permissible Values [#171868635]

Only add permissible values where `is_available` is true to the
`RequestStore` cache. Fixes a regression from SPARCRequest 3.5
where disabled values appear in the protocol form.

ying-pbrc committed:
update messages subject column display

kayla-glick committed:
[#168226099] add All Protocols filter for general users

kayla-glick committed:
[#164840135] epic interface graceful error handling

dependabot-preview[bot] committed:
Bump pdfkit from 0.8.4.1 to 0.8.4.2

Bumps [pdfkit](https://github.com/pdfkit/pdfkit) from 0.8.4.1 to 0.8.4.2.
- [Release notes](https://github.com/pdfkit/pdfkit/releases)
- [Changelog](https://github.com/pdfkit/pdfkit/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pdfkit/pdfkit/commits/0.8.4.2)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2274 from sparc-request/dependabot/bundler/pdfkit-0.8.4.2

Bump pdfkit from 0.8.4.1 to 0.8.4.2

Stuart-Johnson committed:
Merge pull request #2254 from sparc-request/dependabot/bundler/ruby-prof-1.3.1

Bump ruby-prof from 1.3.0 to 1.3.1

Stuart-Johnson committed:
Merge pull request #2256 from sparc-request/dependabot/bundler/parallel_tests-2.32.0

Bump parallel_tests from 2.31.0 to 2.32.0

Stuart-Johnson committed:
Merge pull request #2259 from sparc-request/dependabot/bundler/premailer-rails-1.11.0

Bump premailer-rails from 1.10.3 to 1.11.0

Stuart-Johnson committed:
Merge pull request #2261 from sparc-request/dependabot/bundler/rails-5.2.4.2

Bump rails from 5.2.4.1 to 5.2.4.2

Stuart-Johnson committed:
Merge pull request #2270 from sparc-request/yw-messages_subject_update

update messages subject column display[169276107]

Stuart-Johnson committed:
Merge pull request #2264 from sparc-request/kg-admin_edit_draft_bug

KG - Admin Edit Subject Count Don't Change to Draft

Stuart-Johnson committed:
Merge pull request #2265 from sparc-request/kg-subject_count_max

KG - Max Subject Count Buttons

Stuart-Johnson committed:
Merge pull request #2269 from sparc-request/kg-ssr_resubmission

KG - SSR Resubmission Review Modal

Stuart-Johnson committed:
Merge pull request #2271 from OCTRI/cache-available-permissible-values

HH - Cache Available Permissible Values [#171868635]

Stuart-Johnson committed:
Merge pull request #2272 from sparc-request/kg-dashboard_all_protocols_filter

KG - Add "All Protocols" Protocol Filter for General Users

kayla-glick committed:
spec fixes

kayla-glick committed:
[#172133208] fix service calendar alignment

kayla-glick committed:
specs are annoying

kayla-glick committed:
binding.pry lol

kayla-glick committed:
[#169156333] fix SR submission

kayla-glick committed:
[#170752951] add arm max subject count button tooltip

kayla-glick committed:
[#168670315] remove get a cost estimate button, default status

kayla-glick committed:
merge conflicts

kayla-glick committed:
remove get a cost estimate from defaults.json

Stuart-Johnson committed:
Merge pull request #2251 from sparc-request/kg-duplicate_ssr_bug

KG - Fix Broken SubServiceRequest ssr_id Code

Stuart-Johnson committed:
Merge pull request #2273 from sparc-request/kg-epic_interface_error_handling

KG - Improved Epic Interface Push Error Handling

Stuart-Johnson committed:
Merge pull request #2275 from sparc-request/kg-service_calendar_alignment

KG - Fix Service Calendar "Consolidated Request" Alignment

Stuart-Johnson committed:
Merge pull request #2276 from sparc-request/kg-ssr_resubmission

KG - SSR Resubmission-related Request Submission Bug

Stuart-Johnson committed:
Merge pull request #2277 from sparc-request/kg-subject_count_max

KG - Max Subject Count "Apply to All" Tooltip

Stuart-Johnson committed:
quick fix for migration

Stuart-Johnson committed:
another fix for migration... -.-

Stuart-Johnson committed:
more migration fixes

ying-pbrc committed:
add survey_completion_alert flag to organization form in CM

ying-pbrc committed:
service survey completion alerts

ying-pbrc committed:
update specs

ying-pbrc committed:
removed extra end

ying-pbrc committed:
alert should sent to all relevant super users

dependabot-preview[bot] committed:
Bump database_cleaner from 1.8.3 to 1.8.4

Bumps [database_cleaner](https://github.com/DatabaseCleaner/database_cleaner) from 1.8.3 to 1.8.4.
- [Release notes](https://github.com/DatabaseCleaner/database_cleaner/releases)
- [Changelog](https://github.com/DatabaseCleaner/database_cleaner/blob/master/History.rdoc)
- [Commits](https://github.com/DatabaseCleaner/database_cleaner/compare/v1.8.3...v1.8.4)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

ying-pbrc committed:
udpate survey notification spec

kayla-glick committed:
update migration

SamWord committed:
code clean up

SamWord committed:
Merge branch 'v3.7.0' into saw-oncore-rpe-endpoint

dependabot-preview[bot] committed:
Bump premailer-rails from 1.11.0 to 1.11.1

Bumps [premailer-rails](https://github.com/fphilipe/premailer-rails) from 1.11.0 to 1.11.1.
- [Release notes](https://github.com/fphilipe/premailer-rails/releases)
- [Changelog](https://github.com/fphilipe/premailer-rails/blob/master/CHANGELOG.md)
- [Commits](https://github.com/fphilipe/premailer-rails/compare/v1.11.0...v1.11.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

jleonardw9 committed:
Hiding the merge column again.

jleonardw9 committed:
Fixing the merge sorting

Stuart-Johnson committed:
fixing IE bug with epic questions [#171630866]

jleonardw9 committed:
Leaving the column visible for now.

jleonardw9 committed:
Reverting some test code.

dependabot-preview[bot] committed:
Bump capistrano from 3.12.1 to 3.13.0

Bumps [capistrano](https://github.com/capistrano/capistrano) from 3.12.1 to 3.13.0.
- [Release notes](https://github.com/capistrano/capistrano/releases)
- [Commits](https://github.com/capistrano/capistrano/compare/v3.12.1...v3.13.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump newrelic_rpm from 6.9.0.363 to 6.10.0.364

Bumps [newrelic_rpm](https://github.com/newrelic/rpm) from 6.9.0.363 to 6.10.0.364.
- [Release notes](https://github.com/newrelic/rpm/releases)
- [Changelog](https://github.com/newrelic/rpm/blob/master/CHANGELOG.md)
- [Commits](https://github.com/newrelic/rpm/compare/6.9.0.363...6.10.0.364)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2279 from sparc-request/dependabot/bundler/database_cleaner-1.8.4

Bump database_cleaner from 1.8.3 to 1.8.4

Stuart-Johnson committed:
Merge pull request #2281 from sparc-request/saw-oncore-rpe-endpoint

SAW - OnCore SOAP Endpoint

Stuart-Johnson committed:
Merge pull request #2282 from sparc-request/dependabot/bundler/premailer-rails-1.11.1

Bump premailer-rails from 1.11.0 to 1.11.1

Stuart-Johnson committed:
Merge pull request #2283 from sparc-request/sj-ie_epic_questions_bug

SJ - Fixing IE bug with epic questions

Stuart-Johnson committed:
Merge pull request #2285 from sparc-request/dependabot/bundler/capistrano-3.13.0

Bump capistrano from 3.12.1 to 3.13.0

Stuart-Johnson committed:
Merge pull request #2286 from sparc-request/dependabot/bundler/newrelic_rpm-6.10.0.364

Bump newrelic_rpm from 6.9.0.363 to 6.10.0.364

Stuart-Johnson committed:
Merge pull request #2284 from sparc-request/jl-historical-protocol-merge-refactor

Jl - Historical protocol merge refactor

Stuart-Johnson committed:
cap version bump

ying-pbrc committed:
enqueue service survey alert emails, move text to locale

ying-pbrc committed:
Merge branch 'v3.7.0' into yw-service_survey_complete_alerts

Stuart-Johnson committed:
Merge pull request #2280 from sparc-request/yw-service_survey_complete_alerts

YW- Service Survey Complete Alerts

jleonardw9 committed:
Merged ssr_ids now number from master protocol's next_ssr_id.

jleonardw9 committed:
Adding a check for protocols that do not have ssrs.

ying-pbrc committed:
enable other funding status options

jleonardw9 committed:
Roughing out check for requests split/notify organization.

SamWord committed:
OnCore Endpoint Spec runs and passes locally

SamWord committed:
All gem version besides Savon and its dependencies match v3.7.0

SamWord committed:
Added condition for RPE messages (no flag added yet) with specs

dependabot-preview[bot] committed:
Bump grape from 1.3.1 to 1.3.2

Bumps [grape](https://github.com/ruby-grape/grape) from 1.3.1 to 1.3.2.
- [Release notes](https://github.com/ruby-grape/grape/releases)
- [Changelog](https://github.com/ruby-grape/grape/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ruby-grape/grape/compare/v1.3.1...v1.3.2)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump pry from 0.13.0 to 0.13.1

Bumps [pry](https://github.com/pry/pry) from 0.13.0 to 0.13.1.
- [Release notes](https://github.com/pry/pry/releases)
- [Changelog](https://github.com/pry/pry/blob/master/CHANGELOG.md)
- [Commits](https://github.com/pry/pry/compare/v0.13.0...v0.13.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Stuart-Johnson committed:
Merge pull request #2288 from sparc-request/dependabot/bundler/grape-1.3.2

Bump grape from 1.3.1 to 1.3.2

Stuart-Johnson committed:
Merge pull request #2289 from sparc-request/dependabot/bundler/pry-0.13.1

Bump pry from 0.13.0 to 0.13.1

Stuart-Johnson committed:
manual test for -d email issue

Stuart-Johnson committed:
Merge branch 'v3.7.0' of github.com:sparc-request/sparc-request into v3.7.0

Stuart-Johnson committed:
manual test for -d email issue again

Stuart-Johnson committed:
removing logging on -d for email issue

kayla-glick committed:
merge conflicts

kayla-glick committed:
update migration

kayla-glick committed:
[#171770941] add service header for all calendar editing clarity

kayla-glick committed:
update old migration

kayla-glick committed:
[#171602442] fix SSR bugs

kayla-glick committed:
service calndar wrapping and delimiters [#171770670]

kayla-glick committed:
another migration fix

kayla-glick committed:
hopefully last migration fix

heathharrelson committed:
catalog user rights with lazy_load_ldap [#172300763]

Patch `organizations_controller.rb` to use `Identity::find_or_create`
when adding user rights to the catalog. Fixes error when `lazy_load_ldap`
is enabled.

dependabot-preview[bot] committed:
Bump webdrivers from 4.2.0 to 4.3.0

Bumps [webdrivers](https://github.com/titusfortner/webdrivers) from 4.2.0 to 4.3.0.
- [Release notes](https://github.com/titusfortner/webdrivers/releases)
- [Changelog](https://github.com/titusfortner/webdrivers/blob/master/CHANGELOG.md)
- [Commits](https://github.com/titusfortner/webdrivers/compare/v4.2.0...v4.3.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

kayla-glick committed:
fixes for cached protocol.service_requests and ssr.protocol assignment

jleonardw9 committed:
Adding in a check to see if the requests under 2 protocols can be merged.

SamWord committed:
Completed SOAP Fault errors and specs

kayla-glick committed:
spec/features/surveyor/surveys/user_edits_question_fields_spec.rb:228

SamWord committed:
Minor comment changes

kayla-glick committed:
last spec fixes

kayla-glick committed:
specs are tricky

SamWord committed:
Added local testing interface, DO NOT MERGE

Stuart-Johnson committed:
Merge pull request #2278 from sparc-request/kg-remove_get_a_cost_estimate

KG - Remove "Get a Cost Estimate" Button / Default Status

Stuart-Johnson committed:
Merge pull request #2287 from sparc-request/yw-funding_status_options

Enable Other Funding Status Options[#170883735]

Stuart-Johnson committed:
Merge pull request #2290 from sparc-request/kg-calendar_service_headers

KG - Add Service Name Display For All Calendar Field Editing

Stuart-Johnson committed:
Merge pull request #2292 from sparc-request/kg-service_calndar_wrapping

KG - Service Calendar Cell Wrapping

Stuart-Johnson committed:
Merge pull request #2293 from OCTRI/catalog-lazy-ldap

HH - Error creating catalog user rights with lazy_load_ldap [#172300763]

Stuart-Johnson committed:
Merge pull request #2294 from sparc-request/dependabot/bundler/webdrivers-4.3.0

Bump webdrivers from 4.2.0 to 4.3.0

Stuart-Johnson committed:
Merge pull request #2295 from sparc-request/saw-oncore-rpe-endpoint

SAW - OnCore RPE/CRPC SOAP Endpoint

Stuart-Johnson committed:
Merge pull request #2296 from sparc-request/kg-random_spec_failures

KG - Random Spec Failures

Stuart-Johnson committed:
merge conflicts

Stuart-Johnson committed:
Merge branch 'v3.7.0' into jl-protocol-merge-ssr-ordering

jleonardw9 committed:
Slight refactor of method for determining ineligible merge.

jleonardw9 committed:
Don't need that any more.

jleonardw9 committed:
Merge branch 'jl-protocol-merge-ssr-ordering' of github.com:sparc-request/sparc-request into jl-protocol-merge-ssr-ordering

kayla-glick committed:
[#169210949] allow multiple IRBs per protocol

kayla-glick committed:
[#169210949] multiple IRB cleanup + specs

kayla-glick committed:
merge conflicts

kayla-glick committed:
[#169210949] fix editing primary IRB

kayla-glick committed:
[#169210949] remove console.log

ying-pbrc committed:
fix survey responses resend button bugs

kayla-glick committed:
spec fixes

kayla-glick committed:
[#156705787] limit epic queue push on SR resubmission

kayla-glick committed:
[#169210949] last spec fixes :)

jleonardw9 committed:
If the only service on a request is inactive we still want to see it.

Stuart-Johnson committed:
Merge pull request #2291 from sparc-request/kg-next_ssr_id_bug

KG - Fix Duplicate SSR / Skipped SSR ID Bugs

kayla-glick committed:
Merge branch 'v3.7.0' into kg-epic_resubmission

kayla-glick committed:
[#156705787] slight logic update and new specs

Stuart-Johnson committed:
Merge pull request #2297 from sparc-request/jl-protocol-merge-ssr-ordering

Jl - Protocol merge ssr ordering and new validations

Stuart-Johnson committed:
Merge pull request #2298 from sparc-request/kg-multiple_irbs

KG - Multiple IRBs Per Protocol, Primary and Secondaries

Stuart-Johnson committed:
Merge pull request #2299 from sparc-request/yw-system_satisfaction_survey_resend_button_bugs

Fix Survey Responses Resend Button Bugs

Stuart-Johnson committed:
Merge pull request #2300 from sparc-request/jl-inactice-service-display-bug

Jl - Inactive service display bug

Stuart-Johnson committed:
Merge pull request #2301 from sparc-request/kg-epic_resubmission

KG - Service Request Resubmission Epic Queue Push Logic Change

Stuart-Johnson committed:
small fix

SamWord committed:
Do not import services, only add one default service to protocol.

SamWord committed:
Override Wash Out's SOAP Fault template to include the header OnCore needs

Stuart-Johnson committed:
adding display of other role details to users table [#172030714]

Stuart-Johnson committed:
fix for phone format [#172447401]

Stuart-Johnson committed:
spec fix

Stuart-Johnson committed:
Merge pull request #2308 from sparc-request/sj-phone_format_fix

SJ - Fix for phone format

Stuart-Johnson committed:
Merge pull request #2307 from sparc-request/sj-associated_users_other_role

SJ - Adding "other" role details to users table

Stuart-Johnson committed:
Merge pull request #2306 from sparc-request/saw-oncore-rpe-endpoint

SAW - OnCore SOAP Endpoint Changes

kayla-glick committed:
merge conflicts

kayla-glick committed:
fix merge conflict issue

amcates commented:
@jleonardw9 @Stuart-Johnson I believe this pull request should just be trashed.  It's really old and I think we have moved past this.  Thoughts?

Stuart-Johnson commented:
@amcates Agreed, I was just never sure if this was something that needed to be revisited.
